tensorflow_datasets/image/mnist.py

```diff
@@ -218,6 +221,143 @@ def _info(self):
         citation=_K_MNIST_CITATION,
     )
 
+class EMNISTConfig(tfds.core.BuilderConfig):
+  """BuilderConfig for EMNIST CONFIG."""
+
+  @api_utils.disallow_positional_args
+  def __init__(self, class_number, train_examples, test_examples, **kwargs):
+    """BuilderConfig for EMNIST class number.
+
+    Args:
+      class_number: There are six different splits provided in this dataset. And have
+      different class numbers.
+
+      train_examples, test_examples: So in these have different test and train character
+      numbers.
+
+      **kwargs: keyword arguments forwarded to super.
+    """
+    super(EMNISTConfig, self).__init__(**kwargs)
+    self.class_number = class_number
+    self.train_examples = train_examples
+    self.test_examples = test_examples
+
+
+class EMNIST(MNIST):
+
+  VERSION = tfds.core.Version('1.0.0')
```

Comment:
nit:

here you have version 1.0.0, but in configs: 0.1.1.
Is it on purpose ?

Reply:
in fact I did 1.0, but I think it might be because of version when I get an error. Then I forget and I didn't check it later. Thank you !

Reply:
Updated to version 1.0.1